astro/src/content/quickstarts/quickstart-javascript-express-api.mdx

```diff
@@ -210,12 +210,23 @@ Copy the `token` from the response, which should look like this:
 
 ### Make The Request
 
+<Aside type="note">
+The code is set up to extract the token from either a cookie or the Authorization header so depending on your preference you can replace `--cookie 'app.at=<your_token>'` with `--header 'Authorization: Bearer <your_token>'` when making requests to the API.
```

Comment:
```suggestion
The code is set up to extract the token from either a cookie or the Authorization header so depending on your preference you can replace `--cookie 'app.at=<your_token>'` with `--header 'Authorization: Bearer <your_token>'` when making requests to the API.

If you use a cookie, make sure you store it in a secure, HTTPOnly cookie to avoid exfiltration attacks. See [Storing OAuth Tokens](/articles/oauth/oauth-token-storage) for more information.
```